scripts/docker_boostrap.sh

```diff
@@ -0,0 +1,35 @@
+set -e
+export
+
+export PGPASSWORD=$POSTGRES_ENV_POSTGRES_PASSWORD
+psql -h "$POSTGRES_PORT_5432_TCP_ADDR" \
+     -p "$POSTGRES_PORT_5432_TCP_PORT" \
+     -U "$POSTGRES_ENV_POSTGRES_USER" \
+     -d "$POSTGRES_ENV_POSTGRES_DB" \
+     -c "create extension if not exists postgis; create extension if not exists hstore;"
+
+/usr/bin/wget https://s3.amazonaws.com/metro-extracts.mapzen.com/new-york_new-york.osm.pbf
```

Comment:
Can this be a variable so I could, say, load up San Francisco instead?

How might we support multiple metro extracts?